selinux/bluechi.te

```diff
@@ -16,8 +16,11 @@ init_daemon_domain(bluechi_agent_t, bluechi_agent_exec_t)
 type bluechi_port_t alias hirte_port_t;
 corenet_port(bluechi_port_t)
 
-permissive bluechi_t;
-permissive bluechi_agent_t;
+type bluechi_agent_port_t;
+corenet_port(bluechi_agent_port_t)
+
+gen_tunable(bluechi_port_bind_any, false)
+gen_tunable(bluechi_agent_port_connect_any, false)
```

Comment:
Could you please move those boolean options to the top of the file and add a description for them similar as system has?

https://github.com/fedora-selinux/selinux-policy/blob/85bbb6d1bc69e84ea2440cff647e426b74f1b40a/policy/modules/system/systemd.te#L14